debian_install.md

```diff
@@ -0,0 +1,100 @@
+# Installing st2 on debian/ubuntu
+
+Follow this guide and we'll lead through the basic installation and setup steps.
+
+## Setting up repos and fetching required software
+
+First let's update the system and fetch software which will be required during installation and setup process:
+```shell
+sudo apt-get update
+sudo apt-get install -y wget apache2-utils software-properties-common apt-transport-https sysvinit-utils
+```
+
+Second, add repositories and install st2.
+```shell
+## NB! This step provides instructions for ubuntu trusty
+# Don't forget to use correct distro name!
+export DISTRO=trusty
+
+wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
+
+sudo -E add-apt-repository "deb https://dl.bintray.com/stackstorm/${DISTRO}_staging stable main"
+unset DISTRO
+
+# Update repo and install st2
+sudo apt-get update && sudo apt-get install -y st2bundle
+```
+
+## Install dependent services
+
+St2 uses MongoDB database and RabbitMQ messaging queue, so we need to install this services.
+```
+sudo apt-get install -y mongodb-server rabbitmq-server
+
+# Allow localhost connections to rabbitmq
+echo '[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}]}].' | sudo tee /etc/rabbitmq/rabbitmq.config
```

Comment:
Are you sure it's needed for minimal install?

It works by default without this thing (tested before). Looks like part of advanced config tuning and so on stuff.